python_scripts/replace_template.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
```

Comment:
This didn't work with venv. Suggest changing to `#! /usr/bin/env python`.

Reply:
Can just leave the whole line out =)

Reply:
Sure. You'll have to do `python replace_template.py` instead of `./replace_template.py`.